docs/changelog.md

```diff
@@ -6,6 +6,7 @@ Change Log
 * **Fix**: Don't expand typealiases of function types to `LambdaTypeName`s in `KSTypeReference.toTypeName()`.
 * **Fix**: Small double and float values were set to 0.0 in %L translation (#1919)
 * **Fix**: Fix typealias type argument resolution in KSP2.
+* **Fix**: Fix KT-18706 in CodeWriter.generateImports (#1920)
```

Comment:
Can we change this to be a bit more user friendly so someone reading this would understand what the impact on their code is.

Reply:
I've updated changelog, if new description is still seems off to you, please provide your version